components/DatePicker/DatePicker.jsx

```diff
@@ -47,37 +47,46 @@ export default class DatePicker extends React.Component<Props> {
   }
 
   state = {
-    open: false,
     valid: true,
   }
 
   componentWillMount() {
-    const defaultValue = moment(this.props.defaultValue || "")
-    if(defaultValue.isValid()) this.handleOnChange(defaultValue)
+    this.props.onChange(this.defaultValue())
```

Comment:
why do we need to trigger an `onChange` when the component is about to be mounted?

Reply:
If I recall the bug well, it won't trigger React Form's change event and the default value won't be applied if we don't do it.

Reply:
yeah, looks like this causes the credit application to break

@rafbgarcia can you share the steps to reproduce the bug this was supposed to fix? I feel like there must be an alternative, triggering user input events during mounting like this is somewhat unconventional and creates unexpected side effects.